src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,32 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	gentity_t *tmp = g_entities;
+	int miner = 0;
+	for (int i = 0; i < level.num_entities; i++, tmp++) 
```

Comment:
If you choose to not rewrite as a `ForEntities` loop, then you can also start i = MAX_CLIENTS, to save 64 iterations per call since entityNums 0-63 are reserved for players.